recipes/configs/llama3_2/knowledge_distillation_distributed.yaml

```diff
@@ -100,7 +102,7 @@ log_peak_memory_stats: False
 # Environment
 device: cuda
 dtype: bf16
-enable_activation_checkpointing: False
+enable_activation_checkpointing: False  # True reduces memory
```

Comment:
does KD support optimizer in bwd, activation offloading?

Reply:
Yeah I believe it should be able to